### ISSUE TITLE
Conda install Tensorflow 2.1 with mkl (avx/avx2 support) not working

### ISSUE THREAD

**Issue by gmatalongthewatchtower** [tensorflow/tensorflow#45632]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): tensorflow 2.1 with mkl binary from Anaconda
- TensorFlow version (use command below): 2.1.0
- Python version: 3.7.9 (default, Aug 31 2020, 17:10:11)
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: na
- GPU model and memory: na

**Describe the current behavior**
tensorflow complains that "Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2".

**Describe the expected behavior**
With Intel's mkl version, we shouldn't get this error message. 

**Standalone code to reproduce the issue**
I have tensorflow2.0 with mkl. 

Here's the code:

```
import tensorflow as tf
from tensorflow import keras

input_A = keras.layers.Input(shape=[5], name="wide_input")
input_B = keras.layers.Input(shape=[6], name="deep_input")
hidden1 = keras.layers.Dense(30, activation="relu")(input_B)
hidden2 = keras.layers.Dense(30, activation="relu")(hidden1)
concat = keras.layers.concatenate([input_A, hidden2])
output = keras.layers.Dense(1, name="output")(concat)
model = keras.models.Model(inputs=[input_A, input_B], outputs=[output])

```

Here's the output:
```
2020-12-12 17:06:46.037846: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2020-12-12 17:06:46.040928: I tensorflow/core/common_runtime/process_util.cc:147] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
```

Here's the mkl check from https://software.intel.com/content/www/us/en/develop/articles/intel-optimization-for-tensorflow-installation-guide.html
Code:
```
major_version = int(tf.__version__.split(".")[0])
if major_version >= 2:
   from tensorflow.python import _pywrap_util_port
   print("MKL enabled:", _pywrap_util_port.IsMklEnabled())
else:
   print("MKL enabled:", tf.pywrap_tensorflow.IsMklEnabled()) 
```
Output:

`MKL enabled: True
`

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

Please let me know how I can assist in debugging this issue.

**Comment by gmatalongthewatchtower** [comment]
I am new to github. Can someone please add label "comp:mkl", as referenced at https://software.intel.com/content/www/us/en/develop/articles/intel-optimization-for-tensorflow-installation-guide.html#inpage-nav-5

**Comment by ravikyram** [comment]
@gmatalongthewatchtower 

What is make/model of your cpu?
I suspect your cpu model does not support AVX instructions sets.See [hardware requirements](https://www.tensorflow.org/install/pip?lang=python3#hardware-requirements).Thanks!

**Comment by gmatalongthewatchtower** [comment]
It does have it. i7-10610U CPU @1.8GHz; 4 Cores 8 Logical Processors

**Comment by ravikyram** [comment]
@gmatalongthewatchtower 

Installation issues within the Anaconda environment are tracked in the Anaconda repo.

Could you please submit a new issue using [this link](https://github.com/ContinuumIO/anaconda-issues/issues) and fill in the template, so that the issue can be tracked there. Thanks!

**Comment by gmatalongthewatchtower** [comment]
Isn't this an issue with Tensorflow? I am not sure what Anaconda environment will do here.

**Comment by NeoZhangJianyu** [comment]
@gmatalongthewatchtower 

The warning is not complaining that AVX is not enabled. It is just an informative message saying only for some ops, Intel's oneDNN would be used and AVX or AVX2 would be turned on.

Please refer to #45853

**Comment by ymodak** [comment]
Closing this issue since it is being addressed on the thread tagged above. Thanks!
